cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -348,8 +348,10 @@ install() {
         _install_home="$CARGO_DIST_FORCE_INSTALL_DIR"
         _install_dir="$CARGO_DIST_FORCE_INSTALL_DIR/bin"
```

Comment:
also this looks like another case where we're incorrectly assuming appending `/bin` is correct -- this won't work for HOME_SUBDIR users.